pset-functions.js

```diff
@@ -174,7 +229,22 @@ Use the following test cases to confirm your program meets the success criteria
 console.log('Problem 4:')
 
 // Add your code below this line
+function isPalindrome(word){
+  const wordSplit = word.split("");
+  const wordReverse = wordSplit.reverse();
+  const wordJoin = wordReverse.join("");
+
 
+  if (word === wordJoin){
+    console.log ('Expected Result: true');
```

Comment:
@cbrooks2019 here, you should just `return true` instead of using `console.log()` to print a value.

In general, you want your functions to strictly return a value, which makes them more flexible. Afterwhich, if you choose to print out the return value of the function, you can do so *outside* of the function (which is what you are doing on lines 244 to 247)